library/ssl_tls12_client.c

```diff
@@ -1732,6 +1732,7 @@ static int ssl_parse_server_psk_hint(mbedtls_ssl_context *ssl,
 }
 #endif /* MBEDTLS_KEY_EXCHANGE_SOME_PSK_ENABLED */
 
+
```

Comment:
```suggestion
```
That's really minor and mostly a matter of preference so unless there's something else to fix this can stay as is.
But in case it happens that you fix this, then please add a new line on line 1694 instead of here.